scripts/xdist/prepare_xdist_nodes.sh

```diff
@@ -0,0 +1,15 @@
+python scripts/xdist/pytest_container_manager.py -a up -n ${XDIST_NUM_CONTAINERS} \
```

Comment:
Can you put a shebang here, just to make sure that we are using bash and not sh? Depending on how a Jenkins job invokes a script, it can default to sh